main.go

```diff
@@ -64,8 +66,9 @@ func run() error {
 	if err = bucket.WaitUntilReady(5*time.Second, nil); err != nil {
 		return err
 	}
+	scope := bucket.Scope(scopeName)
+	col := scope.Collection(collectionName)
```

Comment:
What do you think about a fallback case here where, if scope/collection aren't supplied in the config, we use the default collection?

e.g. if scope is nil, don't use it, if collection is nil, get the default collection on the bucket